src/Serializers/SlottySerializer.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Wikibase\DataModel\Serializers;
+
+use Serializers\DispatchableSerializer;
+use Serializers\Exceptions\SerializationException;
+use Wikibase\DataModel\Slotty;
+
+/**
+ * Package private
+ *
+ * @licence GNU GPL v2+
+ * @author Adam Shorland
+ * @since 1.9.0
+ */
+class SlottySerializer implements DispatchableSerializer {
```

Comment:
I wonder why we need a serializer for this if the object's "slots" are just returned as they exist. If serialization of slots should be that simple, just get rid of that serializer and use `$object->getSlots()` directly.